packages/component-library/src/components/plmg-text-input/plmg-text-input.stories.js

```diff
@@ -66,30 +99,26 @@ const Template = (args) => {
 export const Primary = Template.bind({});
 Primary.storyName = 'Text Input';
 Primary.args = {
-  label: 'Text Input',
+  disabled: false,
   ['error-message']: '',
-  ['show-label']: false,
+  label: 'Text Input',
+  ['min-length']: '',
+  ['max-length']: '',
+  name: '',
+  placeholder: '',
+  ['read-only']: false,
   required: false,
+  size: 'medium',
+  ['show-label']: true,
+  tip: '',
   value: '',
+  width: '',
 };
-
-export const AllSizes = (args) => {
-  const htmlContent = sizes
-    .map(
-      (size) =>
-        `<plmg-text-input size=${size} size=${size} label=${size}></plmg-text-input>`
-    )
-    .join('')
-    .trim();
-
-  const el = document.createElement('div');
-  el.innerHTML = htmlContent;
-  el.style.display = 'flex';
-  el.style.justifyContent = 'space-between';
-
-  return el;
+Primary.argTypes = {
+  ['min']: { control: { disable: true } },
```

Comment:
Why disabling them? It prevents me from testing the text-input of type number :confused: